src/directives/ctr-input.ts

```diff
@@ -157,6 +158,19 @@ export class CtrInput {
         }
     }
 
+    @HostListener("click", ["$event"])
+    public onClick(event: any) {
+        this.el.nativeElement.select();
```

Comment:
We already have the property `openOnClick` as default value `false`. The functionality that I implemented will just work if this property is `true`. Take a look on README change that I did. If this property is not good to you, please let me know.